src/classes/container.ts

```diff
@@ -11,35 +11,33 @@
 import { Constants } from "../utils/constants";
 
 export class Container {
-    interactivity: IContainerInteractivity;
-    options: IOptions;
-
-    retina: Retina;
-    canvas: Canvas;
-    particles: Particles;
-
-    checkAnimFrame?: number;
-    drawAnimFrame?: number;
-
-    bubble: IBubble;
-    repulse: IRepulse;
-    svg: ISvg;
-    img: IImage;
-
-    lastFrameTime = 0;
-    pageHidden = false;
+    public interactivity: IContainerInteractivity;
+    public options: IOptions;
+    public retina: Retina;
+    public canvas: Canvas;
+    public particles: Particles;
+    public checkAnimFrame?: number;
+    public drawAnimFrame?: number;
+    public bubble: IBubble;
+    public repulse: IRepulse;
+    public svg: ISvg;
+    public img: IImage;
+    public lastFrameTime: number;
+    public pageHidden: boolean;
 
     constructor(tagId: string, params: IOptions) {
+        this.lastFrameTime = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 0](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)